modules/formulize/class/templateScreen.php

```diff
@@ -84,10 +84,10 @@ function insert($screen) {
         $screen->assignVar('sid', $sid);
 
         if (!$update) {
-            $sql = sprintf("INSERT INTO %s (sid, custom_code, template, donedest, savebuttontext, donebuttontext, viewentryscreen) VALUES (%u, %s, %s, %s, %s, %s, %s)", $this->db->prefix('formulize_screen_template'),
+            $sql = sprintf("INSERT INTO %s (sid, custom_code, template, donedest, savebuttontext, donebuttontext, viewentryscreen) VALUES (%u, '%s', '%s', '%s', '%s', '%s', '%s')", $this->db->prefix('formulize_screen_template'),
```

Comment:
This one is ok because the value being passed in is being put through quoteString